forum/models.py

```diff
@@ -24,6 +25,9 @@ class Forum(models.Model):
     def __str__(self):
         return self.slug
 
+    def get_absolute_url(self):
+        return reverse('forum_detail', args=(self.slug,))
```

Comment:
[get_absolute_url](https://docs.djangoproject.com/en/2.0/ref/models/instances/#get-absolute-url) Allows you to easily link to a Model instance on your site.